src/Sylius/Behat/Context/Api/Shop/AddressContext.php

```diff
@@ -163,6 +172,116 @@ public function iSetTheAddressOfAsDefault(string $fullName): void
         $this->customerClient->update();
     }
 
+    /**
+     * @When /^I try to edit the (address of "([^"]+)")$/
+     */
+    public function iTryToEditTheAddressOf(AddressInterface $address)
```

Comment:
```suggestion
    public function iTryToEditTheAddressOf(AddressInterface $address): void
```
and in the following functions